hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -3227,6 +3227,13 @@ public AltoConfig getAltoConfig() {
         return altoConfig;
     }
 
+    @Beta
+    public Config setAltoConfig(@Nonnull AltoConfig altoConfig) {
```

Comment:
Can we add some documentation for these methods (getter and setter)?

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/57f4a326887f98dfdb6329505b31d651c2376e7c